samples/python/rag/text_rerank.py

```diff
@@ -0,0 +1,31 @@
+#!/usr/bin/env python3
+# Copyright (C) 2025 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import argparse
+import openvino_genai
+
+
+def main():
+    parser = argparse.ArgumentParser()
+    parser.add_argument("model_dir")
+    parser.add_argument("query")
+    parser.add_argument("texts", nargs="+")
+    args = parser.parse_args()
+
+    device = "CPU"  # GPU can be used as well
```

Comment:
Maybe add the device as an argument.

Reply:
The current state intentionally invites people to modify the implementation